### PR TITLE
Move PostAuthor to Post::Author

### PR DESCRIPTION
**Pull request by Marri**
LET THE SORTING CONTINUE

### CODE REVIEW

File: script/seed_dumper.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@
   TagTag: ['created_at', 'updated_at', 'suggested'],
   PostTag: ['created_at', 'updated_at', 'suggested'],
   'Audited::Audit': [],
-  PostAuthor: ['created_at', 'updated_at'],
+  Post::Author: ['created_at', 'updated_at'],
   Message: ['created_at', 'updated_at'],
   Post::View: ['created_at', 'updated_at', 'ignored', 'notify_message', 'notify_email', 'warnings_hidden'],
```
Comment (teceler) [inline script/seed_dumper.rb:21]: These both need `=>`

File: spec/models/post_author_spec.rb [hunk 2]
```diff
@@ -1,4 +1,4 @@
-RSpec.describe PostAuthor do
+RSpec.describe Post::Author do
```
Comment (teceler) [inline spec/models/post_author_spec.rb:1]: According to rubocop this should be moved to spec/models/post/author_spec.rb

File: spec/controllers/posts_controller_spec.rb [hunk 3]
```diff
@@ -2502,16 +2502,16 @@ def restore(reply)
       expect(flash[:success]).to eq("Post deleted.")
     end
 
-    it "deletes PostAuthors" do
+    it "deletes Post::Authors" do
```
Comment (teceler) [inline spec/controllers/posts_controller_spec.rb:2505]: Not sure if this was intended or overeager find/replace but is fine either way, just stood out to me.

Comment (Marri) [reply spec/controllers/posts_controller_spec.rb:2505]: Both, find/replace found it but I did intentionally decide that if the message is specifying the name of the class then I would specify the _correct_ name of the class at least.

**Pull request merged**